### PR TITLE
doc: skipping (obvious) details here is worth making this more nice t…

### PR DESCRIPTION
**Pull request by tshepang**
…o read

### CODE REVIEW

File: src/libcollections/vec.rs [hunk 1]
```diff
@@ -528,7 +528,7 @@ impl<T> Vec<T> {
     }
 
     /// Inserts an element at position `index` within the vector, shifting all
-    /// elements after position `i` one position to the right.
+    /// elements after the position to the right.
```
Comment (brson) [inline src/libcollections/vec.rs]: I found this sentence hard to parse, not realizing "the postion" was referencing that from the previous sentence. What about "shifting all elements after *it* to the right"?

File: src/libcollections/vec.rs [hunk 2]
```diff
@@ -570,7 +570,7 @@ impl<T> Vec<T> {
     }
 
     /// Removes and returns the element at position `index` within the vector,
-    /// shifting all elements after position `index` one position to the left.
+    /// shifting all elements after the position to the left.
```
Comment (brson) [inline src/libcollections/vec.rs]: Ditto.

File: src/libcollections/vec.rs (hunk 1 above)
Comment (tshepang) [inline src/libcollections/vec.rs]: done

File: src/libcollections/vec.rs (hunk 2 above)
Comment (tshepang) [inline src/libcollections/vec.rs]: done
